newsfragments/2211.deprecated.rst

```diff
@@ -0,0 +1,12 @@
+``MultiError`` has been deprecated in favor of the standard :exc:`BaseExceptionGroup`
+(introduced in :pep:`654`). On Python < 3.11, this exception and its derivative
+:exc:`ExceptionGroup` are provided by the backport_. Trio still raises ``MultiError``,
+but it has been refactored into a subclass of :exc:`BaseExceptionGroup` which users
+should catch instead of ``MultiError``. Uses of the ``filter()`` class method should be
+replaced with :meth:`BaseExceptionGroup.split`. Uses of the ``catch()`` class method
+should be replaced with either ``except*`` clauses (on Python 3.11+) or the ``catch()``
+context manager provided by the backport_.
```

Comment:
Please document the differences between the replacements and the old functions.

Reply:
In the release notes, or in the actual documentation?

Reply:
I think an overview in the release notes would be helpful. The documentation explains the new behavior but doesn't particularly contrast it with the old behavior.